### ISSUE TITLE
greenmask restore fails for generated columns

### ISSUE THREAD

**Issue by MamesPalmero** [GreenmaskIO/greenmask#77]
I'm running `greenmask reset --config config.yml latest` and it fails when trying to restore a table with a [generated column](https://www.postgresql.org/docs/current/ddl-generated-columns.html). I'm testing without applying transformations to any of the columns of the table that contains the `generated column` I get the following error message:
`FTL fatal error="data stage restoration error: at least one worker exited with error: unable to perform restoration task (worker 4 restoring table \"public\".\"asdfasdf\"): error from postgres connection msg = column \"state\" is a generated column code=42P10"`

According to the postgres documentation `A generated column cannot be written to directly. In INSERT or UPDATE commands, a value cannot be specified for a generated column, but the keyword DEFAULT may be specified.`, so I have also tried to apply the following transformation:

```yml
dump:
  transformation:
    - schema: 'public'
      name: 'asdfasdf'
      transformers:
        - name: 'Replace'
          params:
            column: 'state'
            value: DEFAULT
```
but I get the same error message. I only get the restore to work if I ignore the table that has a generated column when doing the dump. I don't know if this is a bug or if greenmask simply doesn't support this type of columns. I hope you will give me the answer.

My specs:
- Greenmask 0.1.10
- PostgreSQL 14.11

**Comment by wwoytenko** [comment]
Hi @MamesPalmero! Thank you for reporting. I will explore your issues and get back soon

**Comment by wwoytenko** [comment]
I've checked your case. So I need to exclude generated columns from `COPY` statement. I will fix it as soon as possible.

**Comment by wwoytenko** [comment]
Fixed. You can build the main branch manually by calling `make build` or wait for the next release (expected the next week)

**Comment by wwoytenko** [comment]
Published in v0.1.11
